config/detekt/detekt.yml

```diff
@@ -106,6 +106,7 @@ complexity:
     ignoreDeprecated: false
     ignorePrivate: false
     ignoreOverridden: false
+    ignoreAnnotatedFunctions: ['Preview']
```

Comment:
This attribute is not resolved, can you please check why it is not resolving? IMO this attribute is added in the new version of detekt.